src/main/java/com/idrsolutions/microservice/JPedalServlet.java

```diff
@@ -270,88 +273,99 @@ protected boolean validateRequest(final HttpServletRequest request, final HttpSe
-                                false,
-                                durationTracker);
+                        commandArgs.add("org.jpedal.examples.text.ExtractTextInRectangle");
+                        commandArgs.add(inputPdf.getAbsolutePath());
+                        commandArgs.add(outputDir.getAbsolutePath() + fileSeparator);
                         break;
                     case "wordlist" :
-                        ExtractTextAsWordlist.writeAllWordlistsToDir(
-                                userPdfFilePath,
-                                paramMap.get("password"),
-                                outputDirStr + fileSeparator,
-                                -1,
-                                durationTracker);
+                        commandArgs.add("org.jpedal.examples.text.ExtractTextAsWordlist");
+                        commandArgs.add(inputPdf.getAbsolutePath());
+                        commandArgs.add(outputDir.getAbsolutePath());
                         break;
                     case "structuredText" :
-                        ExtractStructuredText checkForStructure = new ExtractStructuredText(userPdfFilePath);
-                        if (checkForStructure.openPDFFile()) {
-                            Document content = checkForStructure.getStructuredTextContent();
-                            if (content != null && content.hasChildNodes() && content.getDocumentElement().hasChildNodes()) {
-                                ExtractStructuredText.writeAllStructuredTextOutlinesToDir(
-                                        userPdfFilePath,
-                                        paramMap.get("password"),
-                                        outputDirStr + fileSeparator + fileNameWithoutExt + fileSeparator,
-                                        durationTracker,
-                                        null);
-                            } else {
-                                throw new JPedalServletException("File contains no structured content to extract.");
-                            }
-                        } else {
-                            throw new JPedalServletException("Unable to open specified file");
-                        }
+                        commandArgs.add("org.jpedal.examples.text.ExtractStructuredText");
+                        commandArgs.add(inputPdf.getAbsolutePath());
+                        commandArgs.add(outputDir.getAbsolutePath());
                         break;
                 }
                 break;
-            default:
-                throw new JPedalServletException("Unrecognised mode specified: " + mode.name());
-        }
-    }
+                default:
+                    try {
+                        throw new JPedalServletException("Unrecognised mode specified: " + mode.name());
+                    } catch (Exception e) {
+                        throw new RuntimeException(e);
+                    }
```

Comment:
Could we not simplify this to a single Exception thrown?
Either
```suggestion
                    throw new JPedalServletException("Unrecognised mode specified: " + mode.name());
```
or
```suggestion
                    throw new RuntimeException("Unrecognised mode specified: " + mode.name());
```